cmd/crc/cmd/version.go

```diff
@@ -37,5 +37,5 @@ var versionCmd = &cobra.Command{
 }
 
 func runPrintVersion(arguments []string) {
-	fmt.Printf("version: %s+%s\n", crcPkg.GetCRCVersion(), crcPkg.GetCommitSha())
+	fmt.Printf("version: %s-%s+%s\n", crcPkg.GetCRCVersion(), crcPkg.GetBundleVersion(), crcPkg.GetCommitSha())
```

Comment:
you want to be able to debug the version of the bundle used.... but this indicates another issue too,.. the CRC binary is not independent of the actual bundle it can support.

Reply:
So a static version is used... but it is not enforced in the code?

Reply:
@gbraad I didn't get that comment, we had it part of Makefile till now and this is just I made it part of codebase.

Reply:
See https://github.com/code-ready/crc/pull/273#issuecomment-507158507